### PR TITLE
Update translation

### LINKED ISSUE #17: Translation of Plot workbench
- [ ] Add translation infrastructure 
- [ ] Hook in to crowdin 

ref: https://github.com/FreeCAD/Addons/issues/15

### PR DESCRIPTION
**Pull request by hasecilu**
Update.

Draft because using `Plot_Series`, `Plot_Positions` creates an error on `return form.findChild(class_id, name)`

Fix #17

### CODE REVIEW

File: freecad/plot/plotSeries/TaskPanel.py [hunk 1]
```diff
@@ -142,77 +142,77 @@ def widget(self, class_id, name):
         name -- Name of the widget
         """
         mw = self.getMainWindow()
-        form = mw.findChild(QtGui.QWidget, "TaskPanel")
+        form = mw.findChild(QtGui.QWidget, "TaskPanel_plotAxes")
```
Comment (sanguinariojoe) [inline freecad/plot/plotSeries/TaskPanel.py]: Replace `TaskPanel_plotAxes` by `TaskPanel_plotSeries`

File: freecad/plot/resources/ui/TaskPanel_plotPositions.ui [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
- <class>TaskPanel</class>
- <widget class="QWidget" name="TaskPanel">
+ <class>TaskPanel_plotPosition</class>
```
Comment (sanguinariojoe) [inline freecad/plot/resources/ui/TaskPanel_plotPositions.ui]: Replace `TaskPanel_plotPosition` by `TaskPanel_plotPositions`

**Review by sanguinariojoe — COMMENTED** [review]
Yeah, you get the errors because a minor typo. Fix it please and I merge straight away

**Comment by hasecilu** [comment]
The change on size policy is to make translation readable.

image

image

File: freecad/plot/plotSeries/TaskPanel.py (hunk 1 above)
Comment (hasecilu) [reply freecad/plot/plotSeries/TaskPanel.py]: Pfff, how I didn't noticed?

Comment (sanguinariojoe) [reply freecad/plot/plotSeries/TaskPanel.py]: Hahaha, my PRs are always full of this kind of errors...

**Pull request merged**